keybase_proofs/templates/keybase_proofs/profile.html

```diff
@@ -18,7 +18,10 @@
     {% for proof in object_list %}
       {% if proof.is_verified %}
         <tr>
-          <td><a href="https://keybase.io/{{ proof.kb_username }}/sigs/{{ proof.sig_hash }}">{{ proof.kb_username }}</a></td>
+          <td><a href="https://keybase.io/{{ proof.kb_username }}/sigs/{{ proof.sig_hash }}">
+              {{ proof.kb_username }}
+              <img alt="Keybase proof status" src="https://keybase.io/_/api/1.0/sig/proof_badge.svg?domain={{ domain }}&kb_username={{ proof.kb_username }}&username={{ user.username }}&sig_hash={{ proof.sig_hash }}">
```

Comment:
maybe we should make this url `/<username>/proof_badge?....` so it's closer to the `/<username>/sigs/` url which isn't part of the api namespace

Reply:
Would it be /username/proof_badge?kb_username=&domain=&sig-hash=?

Reply:
yeah

Reply:
Yeah, that sounds fine. Might be nicer for people to embed on their webpage too.